src/renderer/entities/wallet/ui/ContactItem/ContactItem.tsx

```diff
@@ -1,45 +1,45 @@
-import { useAddressInfo } from '@entities/wallet/lib/useAddressInfo';
+import { MouseEvent } from 'react';
+
 import { cnTw, toAddress } from '@shared/lib/utils';
-import { Icon, Identicon, BodyText, InfoPopover, HelpText } from '@shared/ui';
-import type { Explorer, AccountId } from '@shared/core';
+import { Identicon, BodyText, HelpText, IconButton } from '@shared/ui';
+import type { Address, AccountId } from '@shared/core';
 
 type Props = {
   name?: string;
-  size?: number;
-  accountId: AccountId;
-  explorers?: Explorer[];
+  address: Address | AccountId;
   addressPrefix?: number;
-  disabled?: boolean;
+  size?: number;
   className?: string;
-  addressFont?: string;
+  onInfoClick?: () => void;
 };
-export const ContactItem = ({
-  accountId,
-  name,
-  size = 20,
-  explorers = [],
-  addressPrefix,
-  disabled,
-  className,
-  addressFont,
-}: Props) => {
-  const address = toAddress(accountId, { prefix: addressPrefix });
-  const popoverItems = useAddressInfo({ address, explorers, addressPrefix, showMatrix: true });
+export const ContactItem = ({ name, address, addressPrefix, size = 20, className, onInfoClick }: Props) => {
+  const formattedAddress = toAddress(address, { prefix: addressPrefix });
+
+  const handleClick = (event: MouseEvent<HTMLDivElement>) => {
+    event.stopPropagation();
+  };
```

Comment:
I think you can just rename it to smth like `stopPropagation` and move it to utils